src/main/java/org/fenixedu/academic/service/services/bolonhaManager/EditCompetenceCourse.java

```diff
@@ -245,16 +246,16 @@ public static void runEditCompetenceCourse(String competenceCourseID, Integer ol
     @Atomic
     public static void runEditCompetenceCourse(String competenceCourseID, String name, String nameEn, Boolean basic,
             CompetenceCourseLevel enumCompetenceCourseLevel, CompetenceCourseType enumCompetenceCourseType,
-            CurricularStage valueOf) throws FenixServiceException {
+            CurricularStage valueOf, String code) throws FenixServiceException {
```

Comment:
This is an API change, the old signature must be kept (deprecated possibly) along with the new one.